Tools/CISupport/ews-build/steps.py

```diff
@@ -213,17 +215,34 @@ def get_reviewers(self, pr_number, repository_url=None):
         if not api_url:
             return []
 
-        reviews_url = f'{api_url}/pulls/{pr_number}/reviews'
-        content = self.fetch_data_from_url_with_authentication_github(reviews_url)
-        if not content:
-            return []
+        reviews = []
+        reviews_url = f'{api_url}/pulls/{pr_number}/reviews?per_page={self.PER_PAGE_LIMIT}'
```

Comment:
do we need to construct this variable here, it's used only to construct the `url` variable. We can directly construct url in one go inside the for loop.

e.g.: `url = f'{api_url}/pulls/{pr_number}/reviews?per_page={self.PER_PAGE_LIMIT}&page={page}`